app/trace/trace_events.ts

```diff
@@ -46,44 +46,60 @@ export type TimeSeriesEvent = TraceEvent & {
 export type TimeSeries = {
   name: string;
   events: TimeSeriesEvent[];
+  unit?: string;
+};
+
+type seriesMetadata = {
```

Comment:
(lowercase typename has no effect in TypeScript)

```suggestion
type SeriesMetadata = {
```

Reply:
Good call